linera-sdk/tests/fixtures/.gitignore

```diff
@@ -0,0 +1 @@
+target/
```

Comment:
Would it be better to add that to the `.gitignore` in the root directory instead?